src/Playwright.Tests/Playwright.Tests.csproj

```diff
@@ -7,21 +7,23 @@
         <NoWarn>1701;1702</NoWarn>
         <AssemblyName>Microsoft.Playwright.Tests</AssemblyName>
         <RootNamespace>Microsoft.Playwright.Tests</RootNamespace>
+        <GenerateProgramFile>false</GenerateProgramFile>
     </PropertyGroup>
     <Import Project="../Common/SignAssembly.props" />
     <ItemGroup>
+		<PackageReference Include="CommandLineParser" Version="2.9.0-preview1" />
```

Comment:
Formatting.

Reply:
I copied it from one file to another, looks like our formatting is inconsistent.

Reply:
Yeah, I think that's VS Mac vs VS inconsistencies.